compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/FirSourceChild.kt

```diff
@@ -38,3 +40,39 @@ private fun FirLightSourceElement.getChild(types: Set<IElementType>, index: Int,
     val visitor = LighterTreeElementFinderByType(treeStructure, types, index, depth)
     return visitor.find(lighterASTNode)?.toFirLightSourceElement(treeStructure)
 }
+
```

Comment:
Now this util file, `FirSourceChild`, should be renamed somehow. :) Say, `FirSourceUtils` ?